dependency-check-maven-config.xml

```diff
@@ -286,6 +286,7 @@
             gwc-web-1.5.0.war: commons-collections-3.1.jar
             gwc-web-1.5.0.war: postgresql-8.4-701.jdbc3.jar
             geowebcache-server-standalone-0.7.0.war: gwc-sqlite-1.9.1.jar
+            geowebcache-server-standalone-0.7.0.war: sqlite-jdbc-3.8.6.jar
```

Comment:
That is an interesting question. The same thing came up with the DDF kernel zip in Alliance, but no issues were reported in DDF. I can't point to the reason why, but I can say that the vulnerabilities are only reported on Alliance, hence the suppression here.